src/Microsoft.DotNet.Build.Tasks/PackageFiles/RunnerTemplate.Windows.txt

```diff
@@ -0,0 +1,74 @@
+﻿@ECHO OFF
+SETLOCAL
+
+SET PACKAGE_DIR=%1
+SET EXECUTION_DIR=%2
+
+IF DEFINED PACKAGE_DIR ( echo Using %PACKAGE_DIR% as folder for resolving package dependencies.) ELSE (
+echo Please specify an package source directory using PACKAGEROOT parameter
+goto ShowUsage
+)
+
+if "%EXECUTION_DIR%" == "" (set EXECUTION_DIR=%~dp0)
+echo Executing in %EXECUTION_DIR% 
+
+:: ========================= BEGIN Test Execution ============================= 
+:RunTests
+
+:: Copy over dependencies... 
+GOTO CopyFiles
+:FileCopySucceeded
+
+set CORE_ROOT=core_root
```

Comment:
Hopefully we don't need this but if we do it should be part of the Test Commands because it will definitely not be needed for anything other then CoreCLR. I would like to eliminate at it though if we can.